### DISCUSSION TITLE
How can I download backups from the server using sftp?

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#1167] [Questions] [answered]
Can I download the backup archives using Filezilla? Or does anyone have a recommendation on how I can pull backups without physical access to my client?

**Comment by szaimen** [comment, marked as answer]
This looks like the wrong approach to me. In case you need to restore something, just restore the backup from the AIO interface which will then automatically get synced to the clients...

**Reply by ghost** [reply]
If I want to transfer a single backup to my computer (useful in case the server hard disk and the backup hard disk break) how can I do this?

**Reply by szaimen** [reply]
I would in this case honestly rather transfer the whole backup repository to your computer as that should not be much bigger due to borgs compression and deduplication. Afterwards you can extract a single archive from the repository by following 
https://github.com/nextcloud/all-in-one#pro-tip-backup-archives-access
